app/assets/main/react_components/footprint_result/YourFootprintText.jsx

```diff
@@ -4,29 +4,33 @@ import { useLocaleData } from '../context/Footprint/LocaleContext.js';
 
 /**
  * Result text for your footprint
- * Includes a heading and the footprint result in tonnes
+ * Includes a footprint result in tonnes
  */
 const YourFootprintText = ({ footprintValue, priceObject }) => {
 
-    const { commonText: { dashboard: { footprint: { heading } }, months: { one: month }, tonnes}, reactContentText: { react: { your_footprint_result_text }}  } = useTexts()
+    const { 
+        commonText: {
+            months: { one: month }, 
+            tonnes
+        }, 
+        reactContentText: {
+            react: { 
+                your_footprint_result_text: { text_part_before, text_part_middle, text_part_end } 
+            }
+        }  
+    } = useTexts();
+
     const {
         currency: {
           money: {
             currency_formats: { [priceObject.currency.iso_code]: currency },
           },
         },
     } = useLocaleData();
-    const [price] = useState(extractPrice())
-    var text = your_footprint_result_text
-    const unit_of_co2 = tonnes
-    const stringKeys = ["%{footprint}" ,"%{unit_of_co2}", "%{price}", "%{time_unit}"]
-    var stringParts = []
 
-    stringKeys.forEach((key) => {
-        stringParts.push(text.split(key)[0])
-        text=text.split(key)[1]
-    })
-    stringParts.push(text)
+    var part_before = text_part_before.replace("%{footprint}","").replace("%{unit_of_co2}","")
+    var part_middle = text_part_middle.replace("%{price}","").replace("%{time_unit}","")
```

Comment:
Tar inte replacesarna bara bort de %{}-delarna som låg i yml-filen? Tänker att du isåfall kan ta bort de därifrån istället? Eller är jag förvirrad nu